requirements.txt

```diff
@@ -0,0 +1,10 @@
+pyinotify==0.9.6
+ipaddress==1.0.17
+PyJWT==1.4.0
```

Comment:
Please take this opportunity to prune these dependencies. I know that PyJWT and sseclient are not used here, they were cut and pasted a long time ago from the marathon controller

Reply:
Where is the dependency on CCCL?

Reply:
Good catch on the unneeded dependencies I will prune them. The CCCL dependency is now in the setup.py file as a dependency_links that will get installed as an install_requires.